.github/workflows/deploy_storybook.yml

```diff
@@ -0,0 +1,24 @@
+name: deploy storybooks
+
+on:
+  push:
+    tags:
+      - production-release
+  workflow_dispatch:
+
+jobs:
+  build_storybooks:
+    name: Build storybooks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 18
+        uses: actions/setup-node@v3
+        with:
+         node-version: 'lts/hydrogen'
+         cache: 'yarn'
+
+      - run: yarn install --production=false --frozen-lockfile
+      - run: yarn workspace @zooniverse/react-components build:es6
+      - run: yarn workspace @zooniverse/classifier build:es6
```

Comment:
Just want to check my understanding of these two lines:

These two lines are building the classifier and lib-react-components, but why build only those two packages when app-project and app-content-pages also have storybooks to deploy? Do we actually need to build packages before deploying their storybooks?

Prior to adding this deploy storybooks action, we would checkout production-release locally and then run `yarn bootstrap` and `yarn deploy-storybook`.

Reply:
Looking at this again, we probably don't need to build the classifier, as it isn't imported into any stories. `@zooniverse/react-components` is imported into stories, though, so that does needs to be built first.

You don't need to build the NextJS apps either, in order to run their storybooks.